package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Visualization components for the web",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "build-harmony": "cd app/harmony && webpack"
```

Comment:
Can we use ``build:harmony`` instead? I feel like the colon "scopes" the build (or other) subcommands better. (If we're ok with the change, it needs to be changed in the Harmony README as well.)